PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -169,6 +153,36 @@ do
 			OnCharacterAdded(character)
 		end)
 	end
+	
+	-- Check for changes in ViewportSize to decide if PortraitMode
+	local CameraChangedConn = nil
+	local function onWorkspaceChanged(property)
```

Comment:
should we check to see if we have touch input? or does this work w/o touch input?